src/brain/operating_unit_recorder.cpp

```diff
@@ -88,7 +89,7 @@ size_t OperatingUnitRecorder::ComputeKeySize(catalog::table_oid_t tbl_oid,
     }
   }
 
-  TERRIER_ASSERT(key_size > 0, "KeySize must be greater than 0");
+  // TERRIER_ASSERT(key_size > 0, "KeySize must be greater than 0");
```

Comment:
Is this assertion still necessary here? I think it can be modified or just removed.

Reply:
Fixed in latest commit, We have to discuss this more with william as this assert is not entirely correct.